icicle/primitives/test.cu

```diff
@@ -406,92 +395,6 @@ TEST_F(PrimitivesTest, INGO_MP_MULT)
   }
 }
 
-TEST_F(PrimitivesTest, INGO_MP_MSB_MULT)
```

Comment:
the primitive tests are important - maybe we can pre-install or replace boost with something more common

Reply:
since we don't need the library for comparing to be CUDA compatible - maybe GMP will work - should be ultimately common on Ubuntu (or at least I can't think of any more common library like that 🙂 ) @DmytroTym  wdyt?

Reply:
@vhnatyk 
Keeping in mind that I am not a member but a user - I understand the fix is to remove tests that don't compile, but this sounds to me like a weak solution. Wouldn't be possible to upgrade the tests so that they are compatible?

Reply:
ok resolving then - otherwise all good @ImmanuelSegol  👍🏻